src/brevitas/export/onnx/manager.py

```diff
@@ -71,7 +71,7 @@ def export_onnx(
 
         if onnx is None or opt is None:
             raise ModuleNotFoundError("Installation of onnx and onnxoptimizer is required.")
-        if not (((input_shape is not None) + (input_t is not None) + (args is not None)) == 1):
+        if not (((input_shape is not None) + (input_t is not None) + (args is not None)) >= 1):
```

Comment:
Keep equality to 1 or things could get weird with passing multiple input definitions